Makefile

```diff
@@ -39,13 +39,21 @@ proto-lint:
 ###############################################################################
 ###                                Initialize                               ###
 ###############################################################################
-start-dev: install
+
+init: kill-dev install 
+	@echo "Initializing both blockchains..."
+	./network/init.sh
+	@echo "Initializing relayer..." 
+	./network/relayer/interchain-acc-config/rly.sh
+
+start: 
+	@echo "Starting up test network"
 	./network/init.sh
 
 start-rly:
 	./network/relayer/interchain-acc-config/rly.sh
 
 kill-dev:
 	@echo "Killing icad and removing previous data"
-	killall icad
 	rm -rf ./data
+	-@killall icad 2>/dev/null
```

Comment:
why switch the order? is there a chance that the process starts writing to that location again and you need to delete all over again?

Reply:
Ah, this line was throwing an error if there was no processes. I'm silencing the error now with the `-` and the `2>/dev/null` but this is the reason.